scripts/asm-installer/asm_vm

```diff
@@ -1462,7 +1462,7 @@ create_gce_instance_template() {
       --format=json \
       --project="${PROJECT_ID}" \
       | jq --arg name "${NEW_INSTANCE_TEMPLATE}" \
-      '.name=$name | .properties.disks[].deviceName=$name' || true)"
+      '.name=$name | .properties.disks[0].deviceName=$name' || true)"
```

Comment:
`'.name=$name | .properties.disks=(.properties.disks | map(if .deviceName==$oldname then .deviceName=$name else . end))'` would be a bit more robust

Reply:
`.deviceName==$oldname` only happens at the first disk and created instance template do not include disks with the same deviceName. I am not sure how that helps.